test/spec/functional/viewActivation.js

```diff
@@ -0,0 +1,242 @@
+var testSrcPath = '../../source',
+    spyOnBackbone = require('./backboneSpy');
+
+describe('A View being deactivated and activated', function() {
+
+  var env, _, $, View, ClickView;
+
+  beforeEach(function(done) {
```

Comment:
still seems like there is a lot of repeated boiler plate for each test.  Could the tests be objects with a test function on them so you could inherit functionality?